### ISSUE TITLE
Bug: Error 600 with only one scene.

### ISSUE THREAD

**Issue by RangerGH** [obsproject/obs-websocket#1151]
### Operating System Info

Windows 11

### OBS Studio Version

29.0.x

### obs-websocket Version

5.1.0

### OBS Studio Log URL

https://obsproject.com/logs/OV9Hgeq8yRq21AXp

### Expected Behavior

If the /scene command is given a scene name and that is the only scene in the list there should be no error thrown but OK returned (or scene already selected).

### Current Behavior

Error 600 thrown when the scene in the /scene command was the only scene in the Scenes list for that collection.

### Steps to Reproduce

1.
2.
3.
...
I am using OBS 29.1.3 and obscommand 1.6.3 and discovered an issue with the /scene command.  I reported this to the obscommand author and  was told to report it here.

1. I have a profile selected say Profile A and then I select a scene collection and finally select a scene with the /scene="MyScene1" command and MyScene1 is the only scene in the Scenes list I get an error 600 of "No source was found by the name of MyScene1".
Always get the error.

2However, if I add a second scene, say Scene 2 to the Scenes list and do /scene="MyScene1" it works without an error.

**Comment by tt2468** [comment]
According to the log you posted, there's no scenes named `MyScene1`. Error code 600 is [`ResourceNotFound`](https://github.com/obsproject/obs-websocket/blob/master/docs/generated/protocol.md#requeststatusresourcenotfound), so that sounds like it's working as intended.

**Comment by RangerGH** [comment]
Yes, I used MyScene1 as an example.  The Scene collection was Elite Dangerous and the only Scene it had was Elite Client.  When I added a second scene, Scene 2, the command worked as it should.

I believe I found the issue.  I had not run the command to set the current scene collection Elite Dangerous so of course it couldn't find it!  Part of my learning curve.  For some reason I can do /scene=xxxx but to choose a Scene Collection I need to run a /command="SetCurrentSceneCollection,sceneCollectionName=yyyy" instead of an equivalent /scenecollection=.

Thanks.
